massa-async-pool/src/pool.rs

```diff
@@ -955,11 +958,11 @@ impl AsyncPool {
         }
 
         // Data
-        if let SetOrKeep::Set(data) = message_update.data {
+        if let SetOrKeep::Set(params) = message_update.function_params {
```

Comment:
Rename (or remove if useless) the comments Handler and Data

Reply:
Done and also in others places in this file.